dev/integration_tests/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -130,6 +131,8 @@ class _CustomThumbShape extends SliderComponentShape {
     SliderThemeData sliderTheme,
     TextDirection textDirection,
     double value,
+    double textScaleFactor,
+    Size sizeWithOverflow,
```

Comment:
I see that it is changed in the API but it should changed here too I think

Reply:
This would require changes to SliderTheme file. Should we continue with it?